### PR TITLE
updated to handle autoconf and version 0.8.0

### PR DESCRIPTION
**Pull request by markllama**
This commit will update rkt.spec so that it builds version 0.8.0-rc1.
This version will build rkt with user-from=host so that it meets Fedora Package Guidelines
The stage1.aci will contain 26 files, all of which were produced solely from sources in the rkt git repo.
This version has also been adapted to use autoconf for configuration.

### CODE REVIEW

File: rkt.spec [hunk 1]
```diff
@@ -34,179 +43,82 @@ BuildRequires: libmount-devel
...
-- New version: 0.5.5, built rkt         commit#b1190d9
+%{_bindir}/%{name}
+%{_libexecdir}/%{name}/stage1.aci
 
-* Wed Apr 29 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.4-3.git40ecb47
-- built rkt commit#40ecb47
+/usr/lib/systemd/system/%{name}-gc.timer
+/usr/lib/systemd/system/%{name}-gc.service
+/usr/lib/systemd/system/%{name}-metadata.socket
+/usr/lib/systemd/system/%{name}-metadata.service
```
Comment (lsm5) [inline rkt.spec]: replace /usr/lib/systemd/system with %{_unitdir}

Also, we can keep the current changelog as-is and have a smaller diff :)

File: rkt.spec [hunk 2]
```diff
@@ -34,179 +43,82 @@ BuildRequires: libmount-devel
...
 
-* Wed Apr 29 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.4-2.git%{d_shortcommit}
-- built rkt commit#40ecb47
+%attr(0755,root,%{name}) %{_sharedstatedir}/%{name}
+%attr(0775,root,%{name}) %{_sharedstatedir}/%{name}/cas
+%attr(0775,root,%{name}) %{_sharedstatedir}/%{name}/cas/db
+%attr(0775,root,%{name}) %{_sharedstatedir}/%{name}/tmp
+%attr(0700,root,%{name}) %{_sharedstatedir}/%{name}/containers
```
Comment (lsm5) [inline rkt.spec]: not sure about group ownership here, is this something that will allow non-sudo users in rkt group to access rkt stuff? kinda like allowing people in docker group to use docker without sudo??

Comment (markllama) [inline rkt.spec]: Will do.

File: rkt.spec [hunk 3]
```diff
@@ -1,31 +1,40 @@
...
+%global builddir %(grep AC_INIT configure.ac  | cut -d' ' -f2| tr -d \]\[,)
+
+%global version %(grep AC_INIT configure.ac  | cut -d' ' -f2| tr -d \]\[, | tr - _)
+
+# valid values: coreos usr-from-src usr-from-host
+%global stage1_usr_from host
+
+Name:       %{repo}
+Version:    %{version}
+Release:    %{shortcommit}%{?dist}
```
Comment (lsm5) [inline rkt.spec]: define Release tag similar to original ... just re-initialize the leading counter to '1' when you have a version bump

**Comment by lsm5** [comment]
could you squash these into 1 commit and force push?

**Comment by lsm5** [comment]
not really a biggie, but just helps me see the full picture :)

File: rkt.spec (hunk 2 above)
Comment (markllama) [inline rkt.spec]: Yeah, that's the purpose. There are operations short of rkt run which are safe for non-root users, but after reading Dan's comments on the docker group, my tendency has changed to requiring sudo.  I'll remove those for now, we can add them back if requested and approved.

File: rkt.spec [hunk 4]
```diff
@@ -34,59 +45,68 @@ BuildRequires: libmount-devel
...
+install -p -m 644 dist/init/systemd/%{name}-gc.service %{buildroot}%{_unitdir}
+install -p -m 644 dist/init/systemd/%{name}-metadata.socket %{buildroot}%{_unitdir}
+install -p -m 644 dist/init/systemd/%{name}-metadata.service %{buildroot}%{_unitdir}
+
+# Install runtime directories
+install -dp %{buildroot}%{_sharedstatedir}/%{name}
+install -dp %{buildroot}%{_sharedstatedir}/%{name}/cas
+install -dp %{buildroot}%{_sharedstatedir}/%{name}/cas/db
+install -dp %{buildroot}%{_sharedstatedir}/%{name}/tmp
+install -dp %{buildroot}%{_sharedstatedir}/%{name}/containers
```
Comment (lsm5) [inline rkt.spec:83]: we can add the dir perms here itself and then skip out the %attr() prepends in %files. i think %attr stuff is deprecated/not needed anymore. everything else lgtm :)

**Pull request merged**